tests/datasets/api_tests.py

```diff
@@ -824,6 +824,8 @@ def test_export_dataset(self):
         Dataset API: Test export dataset
         """
         birth_names_dataset = self.get_birth_names_dataset()
+        if birth_names_dataset.database.backend == "presto":
```

Comment:
maybe add a todo / git issue to fix it ?

Reply:
that's fast @bkyryliuk :)

Still want to see it fail, so doing the `pass`

Reply:
Did some debuging, take a look at `36f0907` commit, where I raise to see the dump from presto metadata. It's really strange it should work.

Reply:
:) try https://github.com/apache/incubator-superset/blob/master/tox.ini#L37 for the local development ?